### DISCUSSION TITLE
Multiple OIDC providers?

### DISCUSSION THREAD

**Discussion by viertaxa** [Budibase/budibase#13376] [Help] [answered]
Hello, 

I'm evaluating Budibase for an admin panel project. There is one hangup that I've run into so far, which is that I can't find a good way to have my apps authenticate against both production and development. The SAML SSO current user token works amazingly well and is pretty much all I need, but I can only have one connection. 

Is there a way to add a second OIDC connection, or is there a better way to achieve this? I'd like to avoid having two instances (using the cloud version) as that would mean we would double up on the licensing cost, especially for the editor seats. 

Thank you!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @viertaxa,

At present, Budibase supports Google Auth along with one OIDC (OpenID Connect). If you're interested in utilising multiple OIDCs, I suggest submitting a feature request on [Github](https://github.com/Budibase/budibase/issues/new/choose).

I'd also recommend reaching out to [sales](https://budibase.com/contact/) to talk through your use case.

Hopefully this answers your question.
